components/raftstore/src/store/peer.rs

```diff
@@ -6220,6 +6241,9 @@ pub trait RequestInspector {
         }
 
         if req.get_header().get_read_quorum() {
+            if req.get_header().get_replica_read() {
+                return Ok(RequestPolicy::ReadIndexReplicaRead);
```

Comment:
The `get_replica_read` seems not tp relate to the `read_quorum` flag, should this check be moved before L6243?